packages/nested-clients/src/nested-sso-oidc/SSOOIDC.ts

```diff
@@ -0,0 +1,67 @@
+// smithy-typescript generated code
+import { createAggregatedClient } from "@smithy/smithy-client";
```

Comment:
The module name already has `nested-` prefix.
I think it can be removed from name of the folder?

Reply:
will remove as part of introducing submodules in this package.